crates/api_models/src/enums.rs

```diff
@@ -519,6 +519,7 @@ pub enum Connector {
     Cybersource,
     #[default]
     Dummy,
+	Bambora,
```

Comment:
run `cargo clippy --all-features --all-targets && cargo +nightly fmt --all` to fix formatting issues